.github/workflows/bench.yml

```diff
@@ -132,15 +156,21 @@ jobs:
               CMD=${CMD//_pacing/--no-pacing}
               EXT="$EXT-nopacing"
             fi
+            CMD=${CMD//_flags/$flags}
           }
 
           # See https://github.com/microsoft/msquic/issues/4618#issuecomment-2422611592
-          for mtu in 1504 65536; do
+          for mtu in 1504; do # We used to measure 65536 here, too, but that's somewhat unrealistic.
```

Comment:
How about removing the loop, given that it iterates at most once. Would reduce the level of indentation.

Reply:
Can do. Was wondering if we wanted to put it back at some point.

Reply:
Should I remove the 65536 MTU everywhere? Would cut down the time on benches.

Reply:
I assume we will focus on Internet paths for now, thus I suggest removing 64k MTU entirely.